geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/+package+_geoportal/__init__.py_tmpl

```diff
@@ -16,7 +17,10 @@ def main(global_config, **settings):
         authentication_policy=create_authentication(settings)
     )
 
+    # Workaround to don't have the error: distutils.errors.DistutilsArgError: no commands supplied
```

Comment:
Workaround to not have the error: distutils.errors.DistutilsArgError: no commands supplied